src/main/java/de/tum/in/www1/artemis/service/dto/OpenAIChatResponseDTO.java

```diff
@@ -0,0 +1,145 @@
+package de.tum.in.www1.artemis.service.dto;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * A DTO representing a response from the OpenAI Chat Completions API.
+ */
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class OpenAIChatResponseDTO {
+
+    @JsonProperty("id")
+    private String id;
+
+    @JsonProperty("object")
+    private String object;
+
+    @JsonProperty("created")
+    private Long created;
+
+    @JsonProperty("model")
+    private String model;
+
+    @JsonProperty("usage")
+    private Usage usage;
+
+    @JsonProperty("choices")
+    private List<Choice> choices;
+
+    public String getId() {
+        return id;
+    }
+
+    public String getObject() {
+        return object;
+    }
+
+    public Long getCreated() {
+        return created;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public Usage getUsage() {
+        return usage;
+    }
+
+    public List<Choice> getChoices() {
+        return choices;
+    }
+
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    public static class Usage {
```

Comment:
It would be good if you add comments to describe the Usage, Choice, and Message classes.